presto-main/src/main/java/com/facebook/presto/sql/planner/iterative/Lookup.java

```diff
@@ -13,43 +13,80 @@
  */
 package com.facebook.presto.sql.planner.iterative;
 
+import com.facebook.presto.Session;
+import com.facebook.presto.cost.CostCalculator;
+import com.facebook.presto.cost.PlanNodeCost;
+import com.facebook.presto.spi.type.Type;
+import com.facebook.presto.sql.planner.Symbol;
 import com.facebook.presto.sql.planner.plan.PlanNode;
 
+import java.util.Map;
 import java.util.function.Function;
 
+import static com.facebook.presto.cost.PlanNodeCost.UNKNOWN_COST;
 import static com.google.common.base.Verify.verify;
 
 public interface Lookup
 {
     /**
      * Resolves a node by materializing GroupReference nodes
      * representing symbolic references to other nodes.
-     *
+     * <p>
```

Comment:
change unrelated to "Change CostCalculator API to use Lookup for computing child costs"